raiden/network/pathfinding.py

```diff
@@ -147,10 +165,10 @@ def get_pfs_info(url: str) -> PFSInfo:
             confirmed_block_number=infos["network_info"]["confirmed_block"]["number"],
             matrix_server=matrix_server_info.netloc,
         )
-    except requests.exceptions.RequestException as e:
+    except RequestException as e:
         msg = "Selected Pathfinding Service did not respond"
         raise ServiceRequestFailed(msg) from e
-    except (json.JSONDecodeError, requests.exceptions.RequestException, KeyError, ValueError) as e:
```

Comment:
We shouldn't handle the `RequestException` here again, right?

Reply:
@konradkonrad ping

Reply:
oh yeah! good find!

Reply:
even though it is just a `noop`